### PR TITLE
Helm charts

### PR DESCRIPTION
**Pull request by fedepaol**
The metrics bit are incomplete as they are still in progress, but the deployment seems to work.

### CODE REVIEW

File: charts/frr-k8s/values.yaml [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  # https://github.com/prometheus/prometheus/blob/release-2.1/documentation/examples/prometheus-kubernetes.yml
+  # for a corresponding Prometheus configuration. Alternatively, you
+  # may want to use the Prometheus Operator
+  # (https://github.com/coreos/prometheus-operator) for more powerful
+  # monitoring configuration. If you use the Prometheus operator, this
+  # can be left at false.
+  scrapeAnnotations: false
+
+  # port both controller and speaker will listen on for metrics
+  metricsPort: 7472
```
Comment (oribon) [inline charts/frr-k8s/values.yaml:24]: nit: leftover comment + we might want to use a different default here than the speakers (I'm thinking of using 7572 in a following metrics pr)

File: charts/frr-k8s/values.yaml [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+  # if set, enables rbac proxy on the controller and speaker to expose
+  # the metrics via tls.
+  # secureMetricsPort: 9120
+
+  # the name of the secret to be mounted in the speaker pod
+  # to expose the metrics securely. If not present, a self signed
+  # certificate to be used.
+  speakerMetricsTLSSecret: ""
```
Comment (oribon) [inline charts/frr-k8s/values.yaml:33]: nit: cleanup some speaker/controller vars

File: charts/frr-k8s/charts/crds/templates/crds.yaml
Comment (oribon) [inline charts/frr-k8s/charts/crds/templates/crds.yaml:1]: is there something that keeps this file synced?

Comment (fedepaol) [reply charts/frr-k8s/charts/crds/templates/crds.yaml:1]: nope (and we have a similar story in metallb)

File: charts/frr-k8s/values.yaml (hunk 1 above)
Comment (fedepaol) [reply charts/frr-k8s/values.yaml]: done

File: charts/frr-k8s/values.yaml (hunk 2 above)
Comment (fedepaol) [reply charts/frr-k8s/values.yaml]: done

File: charts/frr-k8s/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+apiVersion: v2
+name: frrk8s
```
Comment (oribon) [inline charts/frr-k8s/Chart.yaml:2]: does it matter frrk8s / frr-k8s here?

File: Makefile [hunk 4]
```diff
@@ -184,6 +197,12 @@ $(KIND): $(LOCALBIN)
 	test -s $(LOCALBIN)/kind && $(LOCALBIN)/kind --version | grep -q $(KIND_VERSION) || \
 	GOBIN=$(LOCALBIN) go install sigs.k8s.io/kind@$(KIND_VERSION)
 
+.PHONY: helm
+helm: $(HELM) ## Download helm locally if necessary. If wrong version is installed, it will be overwritten.
+$(HELM): $(LOCALBIN)
+	test -s $(LOCALBIN)/helm && $(LOCALBIN)/helm version | grep -q $(HELM_VERSION) || \
+	USE_SUDO=false HELM_INSTALL_DIR=./bin DESIRED_VERSION=$(HELM_VERSION) bash <(curl -s https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3)
```
Comment (oribon) [inline Makefile:205]: `./bin` -> `$(LOCALBIN)` ?

File: charts/frr-k8s/Chart.yaml (hunk 3 above)
Comment (fedepaol) [reply charts/frr-k8s/Chart.yaml]: done

File: Makefile (hunk 4 above)
Comment (fedepaol) [reply Makefile]: done

**Review by oribon — APPROVED** [review]
lgtm

**Pull request merged**